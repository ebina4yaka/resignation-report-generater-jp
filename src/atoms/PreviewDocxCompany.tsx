import React from 'react'
import { createStyles, makeStyles } from '@material-ui/core/styles'
import {
  getCompanyName,
  getRepresentativeDirector,
} from '../libs/export/textTemplates'

type Props = {
  companyName: string
  representativeDirector: string
}

const useStyles = makeStyles(
  createStyles({
    root: {
      textAlign: 'left',
    },
  })
)

export default function PreviewDocxCompany(props: Props): React.ReactElement {
  const { companyName, representativeDirector } = props
  const classes = useStyles()
  return (
    <div className={classes.root}>
      <div>{getCompanyName(companyName)}</div>
      <div>{getRepresentativeDirector(representativeDirector)}</div>
    </div>
  )
}
