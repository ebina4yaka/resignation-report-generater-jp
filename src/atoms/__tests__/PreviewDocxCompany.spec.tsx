import React from 'react'
import renderer from 'react-test-renderer'
import PreviewDocxCompany from '../PreviewDocxCompany'

test('Snapshot test for PreviewDocxCompany', () => {
  const tree = renderer
    .create(
      <PreviewDocxCompany
        companyName="株式会社無職"
        representativeDirector="無職社長"
      />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
