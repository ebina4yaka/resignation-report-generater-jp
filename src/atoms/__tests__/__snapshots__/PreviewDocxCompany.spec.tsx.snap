// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for PreviewDocxCompany 1`] = `
<div
  className="makeStyles-root-1"
>
  <div>
    株式会社無職 御中
  </div>
  <div>
    （代表取締役 無職社長 様）
  </div>
</div>
`;
